ontology/ado.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: ontology_detail
+id: ado
+title: Alzheimer's Disease Ontology
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
+  label: Alpha Tom Kodamullil
+  github: akodamullil
+  orcid: 0000-0001-9896-3531
+description: Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer'S Disease, foundamentally structured by upper level Basic Formal Ontology(BFO). This Ontology is enriched by the interrelational entities that demonstrate the nextwork of the understanding on Alzheimer's disease and can be readily applied for text mining.
+domain: health
+homepage: https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO
+products:
+  - id: ado.owl
+dependencies:
+  - id: bfo
+tracker: https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO/issues
+license:
+  url: https://creativecommons.org/licenses/by/4.0/
+  label: CC BY 4.0
+repository: https://github.com/Fraunhofer-SCAI-Applied-Semantics/ADO
+preferredPrefix: ADO
+---
+
+Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer'S Disease, foundamentally structured by upper level Basic Formal Ontology(BFO). This Ontology is enriched by the interrelational entities that demonstrate the nextwork of the understanding on Alzheimer's disease and can be readily applied for text mining.
```

Comment:
various typos:

- foundamentally is not a word
- interrelational is not a word (interrelated entities)
- nextwork is probably supposed to be network?

Reply:
Thank you for your feedback! The types are now corrected.

Reply:
Great!